ui/ui-components/pages/model/[modelId]/source/[sourceId]/edit.tsx

```diff
@@ -28,22 +29,24 @@ import PrimaryKeyBadge from "../../../../../components/badges/PrimaryKeyBadge";
 import { useApi } from "../../../../../contexts/api";
 import { generateClient } from "../../../../../client/client";
 import { withServerErrorHandler } from "../../../../../utils/withServerErrorHandler";
+import { SourcePreviewMethodResponseRow } from "@grouparoo/core/src/classes/plugin";
 
-interface FormData {
+export interface FormData {
   mapping?: {
     sourceColumn: string;
     propertyKey: string;
   };
-  source: Pick<Models.SourceType, "name" | "options">;
+  source: Pick<Actions.SourceView["source"], "name" | "options">;
```

Comment:
is `Actions.SourceView["source"]` different from `Models.SourceType`?

Reply:
Yes

Reply:
Or not? I had errors mixing them previously.